datasets/drenthe/time_curves/energy_distribution_woody_biomass_time_curve.csv

```diff
@@ -0,0 +1,83 @@
+year,max_demand
```

Comment:
I fixed the broken time curves, it was indeed an issue with the scaling script. You can grab the fixed curves by running `git pull`.